Main.java

```diff
@@ -90,32 +221,35 @@ public Parallelogram (float b, float h){
     else
       height = h;
   }
-  public float Area(){
+  public float area(){
     return base*height;
   }
 
 }
+
+//Quadilateral class
 class Quadilateral {
   private float length;
   private float breadth;
 
   public void setLength(float l){
-    length = l;
+   this.length = l;
   }
 
   public float getLength(){
-    return length;
+    return this.length;
   }
 
   public void setBreadth(float b){
-    breadth = b;
+    this.breadth = b;
   }
 
   public float getBreadth(){
-    return breadth;
+    return this.breadth;
   }
 
   public float calculatePerimeter(){
     return 2*(length+breadth);
   }
+
```

Comment:
your code is very similar to zoe, ryan, and Subomi codes. Just that your switch case is different. Still wondering who copied who or if all of you chose to do it as a group assignment when I gave it as an individual assignment